### PR TITLE
Updated Readme

### PR DESCRIPTION
**Pull request by jasvinderkhurana**
Updated Readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
-Kria SOM Base HW Projects
+<h1 align="center">KRIA SOM BASE HARDWARE DESGIN</h1>
 
-To generate an XSA for starter kits
+## Introduction
+This repository contains the Kria SOM base hardware projects. It contains varous tcl file which can be used to generate XSA for the required starter kit.
```
Comment (chkohn) [inline README.md]: @jasvinderkhurana typo s/varous/various/ and files should be plural

**Review by chkohn — COMMENTED** [review]
please see inline comment

File: README.md (hunk 1 above)
Comment (jasvinderkhurana) [reply README.md]: updated! please merge the pull request.

**Comment by chkohn** [comment]
Merged via 0ca3ec4403a451519a5fe8d0ee6a0750f49a7931

**Pull request closed without merging**